sql/rpl_slave.cc

```diff
@@ -7897,29 +7611,13 @@ wsrep_restart_point :
     The thd can only be destructed after indirect references
     through mi->rli->info_thd are cleared: mi->rli->info_thd= NULL.
 
-<<<<<<< HEAD
     For instance, user thread might be issuing show_slave_status
     and attempting to read mi->rli->info_thd->get_proc_info().
     Therefore thd must only be deleted after info_thd is set
     to NULL.
   */
-  delete thd;
-||||||| 35582423e36
-      For instance, user thread might be issuing show_slave_status
-      and attempting to read mi->rli->info_thd->get_proc_info().
-      Therefore thd must only be deleted after info_thd is set
-      to NULL.
-    */
-    delete thd;
-=======
-      For instance, user thread might be issuing show_slave_status
-      and attempting to read mi->rli->info_thd->get_proc_info().
-      Therefore thd must only be deleted after info_thd is set
-      to NULL.
-    */
     mysql_thread_set_psi_THD(nullptr);
-    delete thd;
->>>>>>> Percona-Server-8.0.25-15
+  delete thd;
```

Comment:
alignment error

Reply:
The alignment issue is in the line above, which isn't changed in this commit. Anyway, I fixed it.